### PR TITLE
Assign-user action initial release

### PR DESCRIPTION
**Pull request by paulbreuler**
tagging to the separate work item: [AB#2871196](https://dev.azure.com/dynamicscrm/1fb98997-2f9e-4734-be8a-9728680447c2/_workitems/edit/2871196)

### CODE REVIEW

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -78,6 +79,15 @@ jobs:
       run: |
         exit 1
 
+    - name: Test assign-user action without role assignment
+      uses: ./assign-user
+      with:
+        user-name: ${{ env.WF_USERNAME }}
+        password-secret: ${{ secrets.PASSWORD_PPDEVTOOLS }}
+        environment: ${{ steps.create-environment.outputs.environment-url }}
+        user: ${{ env.WF_USERNAME}}
```
Comment (davidjenni) [inline .github/workflows/pull-request.yml:88]: should we use a separate user?

File: assign-user/action.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+  client-secret:
+    description: 'The client secret to authenticate with. Required if authenticating with app-id.'
+    required: false
+
+  tenant-id:
+    description: 'Tenant id if using app-id & client secret to authenticate.'
+    required: false
+
+  cloud:
```
Comment (davidjenni) [inline assign-user/action.yml:39]: remove for now, should be addressed with AB#2871196

**Review by davidjenni — DISMISSED** [review]

File: assign-user/action.yml (hunk 2 above)
Comment (davidjenni) [reply assign-user/action.yml:39]: tagging to the separate work item: AB#2871196

Comment (paulbreuler) [reply assign-user/action.yml:39]: Darn thought I did. Must have pruned it from the other PR and not this one.

File: .github/workflows/pull-request.yml (hunk 1 above)
Comment (paulbreuler) [reply .github/workflows/pull-request.yml:88]: We can grab the ID of another user to test with to ensure the API call to sync is fully functional at this level.

Comment (paulbreuler) [reply .github/workflows/pull-request.yml]: This has been updated, per your request.

**Review by petrochuk — APPROVED** [review]

**Pull request merged**